examples/1.0.0/oauth.workflow.yaml

```diff
@@ -52,9 +52,12 @@ workflows:
         description: >-
           This is where you do the refresh
         operationId: get-token
-        parameters:
-          grant_type: refresh_token
-          refresh_token: $steps.do-the-auth-flow.outputs.my_refresh_token
+        requestBody:
+          payload:
+            contentType: application/x-www-form-urlencoded
+            payload:
```

Comment:
Do we want payload: again here? Not sure if Body might be better? Not bothered either way just noticing payload twice.